apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -2317,7 +2317,7 @@ suite('thread_list_ui', function() {
 
       Drafts.on.withArgs('saved').yield(threadDraft);
 
-      assert.isTrue(InboxView.notifyAboutSavedDraft);
+      assert.equal(InboxView.notifyAboutSavedDraftWithId, threadDraft.id);
```

Comment:
I think the same, obviously :) I just didn't want to change tests too much, but since you're OK with improving all tests that rely on this internal detail - I'll fix them!